src/api/storage.ts

```diff
@@ -58,6 +56,7 @@ const getOptionsStorage = (): MultipleOptions => {
       return courseOptions
 
     } else {
+      console.log('Invalid storage')
```

Comment:
Maybe this console.log as well, since it is not relevant to the user that the storage is invalid, since the application will self heal by resetting the localstorage.

Reply:
done!